ozkour-front/src/components/EventArray.vue

```diff
@@ -6,14 +6,80 @@
       <table>
         <tr>
           <th scope="col"></th>
-          <th scope="col">DATE</th>
-          <th scope="col">UNIVERS</th>
-          <th scope="col">TYPE</th>
-          <th scope="col">NOM DE L'EVENEMENT</th>
-          <th scope="col">TITRE DU TALK</th>
-          <th scope="col">SPEAKER</th>
+          <th
+            scope="col"
+            :class="{ columnSelected: sort === 'date', selectable: true }"
+            v-on:click="setSort('date')"
+          >
+            DATE
+            <div
+              class="arrow"
+              v-if="sort === 'date'"
+              v-bind:class="ascending ? 'arrow_up' : 'arrow_down'"
```

Comment:
```suggestion
              :class="ascending ? 'arrow_up' : 'arrow_down'"
```

Reply:
Alors pour l'image, c'est mieux d'avoir un svg au lieu d'un png (beaucoup plus lourd et pas très responsive).
https://kinsta.com/fr/blog/svg-vs-png/
https://blog.logrocket.com/using-svg-and-vue-js-a-complete-guide/